bgpd/bgp_evpn.c

```diff
@@ -4268,8 +4272,18 @@ static int process_type5_route(struct peer *peer, afi_t afi, safi_t safi,
 	 * field
 	 */
 
+	if (attr) {
+		is_valid_update = true;
+		if (is_zero_mac(&attr->rmac) && is_zero_esi(&evpn.eth_s_id) &&
```

Comment:
looks like formatting is not as it should be

Reply:
checkpatch.sh was run on this change and it passed.